tests/test-docs-source/test.rst

```diff
@@ -279,6 +279,29 @@ Autodoc + order by source:
    :members:
    :undoc-members:
 
+.. note::
+
+   .. autoclass:: ExampleInheritedSubclass
+      :members:
+      :undoc-members:
+      :inherited-members:
+
+   .. code-block:: none
+
+      <code class="(sig-name )?descname">a_syncmethod</code>
+
+   .. code-block:: none
+
+      <code class="(sig-name )?descname">b_syncmethod</code>
```

Comment:
Is this intended to check that they're being rendered as sync functions? Unfortunately I don't think it accomplishes that, because that regex will match regardless of whether there's an `await` earlier on the line.

You could extend the regex to match some extra text before the initial `<code`, or you could use a "negative test", where we assert that the regex does *not* match. Unfortunately you can't do this on a regex-by-regex basis; instead, you do it by using a `.. warning::` block instead of a `.. note::` block. So you'd need to remove these here, and then add something below like:

```
.. warning::

   .. autoclass:: ExampleInheritedSubclass

   .. code-block:: none

      <em class="property">await </em><code class="(sig-name )?descname">a_syncmethod</code>

   .. code-block:: none

      <em class="property">await </em><code class="(sig-name )?descname">b_syncmethod</code>
```

Reply:
Ah, I see. Changed in the next commit.